src/App/Options/Cmd/SqsToKafka.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE DeriveAnyClass   #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE TemplateHaskell  #-}
+
+module App.Options.Cmd.SqsToKafka
```

Comment:
What's this command going to be?

Reply:
I'm planning to write something that reads S3 events extracts just the bits we want and put it in Kafka.